packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -126,6 +126,7 @@ EEWorkflowController.post(
 		Object.assign(newWorkflow, req.body);
 
 		newWorkflow.versionId = uuid();
+		newWorkflow.settings = { ...newWorkflow.settings, executionOrder: 'v1' };
```

Comment:
Is this necessary? Should not the front end already send it already correctly? Seems strange that if the workflow-create-request we receive would particularly mention creating it for `v0` we would overwrite it.

Reply:
I originally couldn't get frontend to send the payload correctly, and added this code.
but then managed to fix this in the frontend, and forgot to revert this.